pallets/subtensor/src/subnets/serving.rs

```diff
@@ -86,6 +90,15 @@ impl<T: Config> Pallet<T> {
             Error::<T>::ServingRateLimitExceeded
         );
 
+        // Check certificate
```

Comment:
We should be cleaning up stored certificates when the neuron is deregistered, there needs to be a matching deletion for this insertion.